js/src/components/app-select-control/index.js

```diff
@@ -12,18 +14,60 @@ import './index.scss';
 /**
  * Renders a `@wordpress/components`'s `SelectControl` with margin-bottom removed.
  *
- * If you provide `className` via props,
- * it will be added to the container div's `className`,
- * so that you can further control its style.
- *
- * @param {*} props
+ * @param {*} props The component props.
+ * @param {Array} props.options Array of options for the select dropdown. Each option should be an object containing `label` and `value` properties.
```

Comment:
The JSDoc of `props.options` was not updated accordingly.

Ref: https://github.com/woocommerce/google-listings-and-ads/pull/2608/commits/08b16e067128362d3053e50fb1e8c7c7ebf9f204#diff-52e887fd2c4ad94dd408eb3141de68f1a16cd7c24e7ab6122f62bb7b52747844R27

Reply:
Updated @eason9487